hazelcast/src/test/java/com/hazelcast/test/AbstractHazelcastClassRunner.java

```diff
@@ -76,17 +78,16 @@ public static String getTestMethodName() {
     }
 
     /**
-     * Creates a BlockJUnit4ClassRunner to run {@code klass}
+     * Creates a BlockJUnit4ClassRunner to run {@code classType}
      *
      * @throws org.junit.runners.model.InitializationError if the test class is malformed.
      */
-    public AbstractHazelcastClassRunner(Class<?> klass) throws InitializationError {
-        super(klass);
```

Comment:
classType is not an improvements over klass. With klass it is at at least obvious that a 'class' is expected, but that the name is in use. classType is going to be the type of a class? So either use klass or clazz..